### ISSUE TITLE
Drafts - delete Method not Available

### ISSUE THREAD

**Issue by vishnuinfrrd** [nylas/nylas-java#6]
Hi,

Below code is documented in Nylas API doc. However, we are not able to use this.

```
NylasClient nylas = new NylasClient();
NylasAccount account = nylas.account("{ACCESS_TOKEN}");
Drafts drafts = account.drafts();
// Delete drafts by specifying the appropriate id
drafts.delete("{draft_id}");
```

We get error when we try tp access `drafts.delete()` method.
> The method delete(String) from the type RestfulDAO<Draft> is not visible
> delete(java.lang.String,java.util.Map<java.lang.String,java.lang.String>) has protected access in com.nylas.RestfulDAO

**Comment by ddlatham** [comment]
@vishnuinfrrd Looks like we missed support for this in the SDK.  Will add to the next release available soon.

**Comment by ddlatham** [comment]
Fixed in release 1.3.0
